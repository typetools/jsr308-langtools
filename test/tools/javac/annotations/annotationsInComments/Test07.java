/*
 * @test
 * @bug 1234567
 * @summary Annotations in comments are recognized
 * @author Werner Dietl
 * @compile/fail/ref=Test07.out -XDrawDiagnostics Test07.java
 */

// Recognized comment, unknown annotation
/*>>>
@Unknown("HiThere!")
*/
class Test07 {}
