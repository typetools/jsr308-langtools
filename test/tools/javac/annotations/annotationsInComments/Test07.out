Test07.java:11:2: compiler.err.cant.resolve: kindname.class, Unknown, , 
1 error